### DISCUSSION TITLE
Why is `package` run twice during CI build?

### DISCUSSION THREAD

**Discussion by automartin5000** [projen/projen#3308] [Q&A] [answered]
As somewhat of a follow up to [this discussion](https://github.com/projen/projen/discussions/3298), I'm trying to understand why it seems like the build workflow runs `package` twice (and then `package` is run _again_ on `Release`). 

I can imagine that maybe package is run once during build as a test to ensure it builds correctly (is that assumption accurate?), but why is it run once under [projen build](https://github.com/projen/projen/blob/7248ab037c51ecec1f77feeef0be4f18e2d59f9a/.github/workflows/build.yml#L29) and then again once for each language in parallel under [projen package:language](https://github.com/projen/projen/blob/7248ab037c51ecec1f77feeef0be4f18e2d59f9a/.github/workflows/build.yml#L112)?

**Comment by automartin5000** [comment, marked as answer]
Never mind, I see the `CI: "true"` env var that skips packaging. Coincidentally, I found it when I had to replace `rsync` in `package` since I can't install `rsync` in my CI environment 🙃

**Reply by mrgrain** [reply]
Yes, it's a horrible implementation currently. High priority to be fixed.
